libraries/botframework-connector/package.json

```diff
@@ -27,6 +27,7 @@
     }
   },
   "dependencies": {
+    "@azure/identity": "2.0.0-beta.5",
```

Comment:
We try to not use `beta` packages - can we use the latest non-beta version on NPM in the `1.x.x` version?

Reply:
Our intention was to use the `1.5.2` version, but we encountered an error with `keytar` when building the project. We found an issue saying that `keytar` will be an optional dependency in `2.0.0`. Therefore, we updated the `@azure/identity` to the `2.0.0-beta` version to be able to build the project.
More information can be found in the **Troubleshooting** section on the PR description.